packages/opentelemetry-exporter-collector/src/platform/node/util.ts

```diff
@@ -46,16 +48,10 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
       'Content-Type': contentType,
       ...collector.headers,
     },
+    agent: collector.agent,
```

Comment:
Not sure if it can really happen. The url is parsed on each invocation which at least looks like it can change.
Therefore we may switch between http/https which requires different agents.

Reply:
url is defined as readonly [here](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-collector/src/CollectorExporterBase.ts#L34)
```js
  public readonly url: string;
```

Another option for performance improvement in my opinion (not for this PR) is to parse the URL once in the constructor instead of doing it on every export.